src/planning/task_planner/scripts/task_planner.py

```diff
@@ -119,177 +120,58 @@ def stateUpdateCallback(self):
-        rospy.loginfo("Received input")
-        self.bot_state.update_hri(msg.speech, Emotions.NEUTRAL)
-
-        if msg.primitive == 'video_call':
-            rospy.loginfo("Updating operation mode to video call")
-            self.bot_state.update_operation_mode(OperationModes.TELEOPERATION)
-        elif msg.primitive == 'stop_video':
-            rospy.loginfo("Updating operation mode to autonomous")
-            self.bot_state.update_operation_mode(OperationModes.AUTONOMOUS)
-        elif msg.primitive != 'do_nothing':
-            self.navigationGoal, self.objectOfInterest = self.mappings[msg.primitive]
-            self.executeTask()
-
-        return GlobalTaskResponse(success = True)
-        
-
     def navigate_to_location(self, location : Enum):
-        locationName = location.name
-        rospy.loginfo(f"[{rospy.get_name()}]:" +"Executing task. Going to {}".format(locationName))
+        #locationName = location.HOME
+        #rospy.loginfo(f"[{rospy.get_name()}]:" +"Executing task. Going to {}".format(locationName))
         # send goal
-        print("Sending goal to move_base", self.goal_locations[locationName]['x'], self.goal_locations[locationName]['y'], self.goal_locations[locationName]['theta'])
+        #print("Sending goal to move_base", self.goal_locations[locationName]['x'], self.goal_locations[locationName]['y'], self.goal_locations[locationName]['theta'])
         goal = MoveBaseGoal()
         goal.target_pose.header.frame_id = "map"
-        goal.target_pose.pose.position.x = self.goal_locations[locationName]['x']
-        goal.target_pose.pose.position.y = self.goal_locations[locationName]['y']
-        quaternion = get_quaternion(self.goal_locations[locationName]['theta'])
+        goal.target_pose.pose.position.x = -3.32
+        goal.target_pose.pose.position.y = -9.54
+        quaternion = get_quaternion(240)
         goal.target_pose.pose.orientation = quaternion
         self.navigation_client.send_goal(goal, feedback_cb = self.bot_state.navigation_feedback)
         wait = self.navigation_client.wait_for_result()
 
         if self.navigation_client.get_state() != actionlib.GoalStatus.SUCCEEDED:
-            rospy.loginfo(f"[{rospy.get_name()}]:" +"Failed to reach {}".format(locationName))
+            rospy.loginfo(f"[{rospy.get_name()}]:" +"Failed to reach Table")
             # cancel navigation
             self.navigation_client.cancel_goal()
             return False
         
-        rospy.loginfo(f"[{rospy.get_name()}]:" +"Reached {}".format(locationName))
+        rospy.loginfo(f"[{rospy.get_name()}]:" +"Reached Table")
         self.bot_state.update_emotion(Emotions.HAPPY)
         self.bot_state.update_state()
         self.bot_state.currentGlobalState = GlobalStates.REACHED_GOAL
-        rospy.loginfo(f"[{rospy.get_name()}]:" +"Reached {}".format(locationName))
+        rospy.loginfo(f"[{rospy.get_name()}]:" +"Reached Table")
```

Comment:
will be better to keep this modular so that we can have accurate logs instead of hardcoding